server-phoenix/lib/helios/announcement.ex

```diff
@@ -19,7 +19,7 @@ defmodule Helios.Announcement do
     field :people, :string
     field :company, :string
     field :announcement_id, :string
-    timestamps(inserted_at: :created_at, type: :utc_datetime)
+    timestamps(inserted_at: :inserted_at, type: :utc_datetime)
```

Comment:
Since the default timestamp in Phoenix is inserted_at, and the default type is :utc_datetime, could we just use `timestamps()` here instead?